layouts/api/list.html

```diff
@@ -368,37 +369,87 @@ <h3 class="mb-2">Code Example</h3>
 
                             <div class="code-snippet-wrapper js-code-snippet-wrapper" >
                                 <ul class="nav nav-tabs border-none response-toggle">
-                                    
                                     {{ $count := 0}}
+                                    {{ $lang := index $dot.Site.Data.highlighting.languages "sh" }}
+
+                                    <li class="nav-item">
+                                        <a class="nav-link mr-1 js-code-example-link text-capitalize {{ if eq $count 0}} active {{end}}" data-code-lang="{{ $lang.pygments_name }}"  href="#" data-toggle="tab">{{ $lang.pygments_name }}</a>
+                                    </li>
+
+                                    {{ $count := 1}}
                                     {{ range . }}
 
                                     {{ $file_extension := index (split (path.Ext .) ".") 1 }}
                                     {{ $lang := index $dot.Site.Data.highlighting.languages $file_extension }}
 
+                                    {{ if ne $file_extension "sh"}}
                                     <li class="nav-item">
                                         <a class="nav-link mr-1 js-code-example-link text-capitalize {{ if eq $count 0}} active {{end}}" data-code-lang="{{ $lang.pygments_name }}"  href="#" data-toggle="tab">{{ $lang.pygments_name }}</a>
                                     </li>
 
                                     {{ $count = add $count 1 }}
+                                    {{ end }}
 
                                     {{ end }}
 
                                 </ul>
 
-                                {{ $count := 0}}
+                                {{ $count := 0 }}
+                                {{ $file_extension := "sh" }}
+                                {{ $lang := index $dot.Site.Data.highlighting.languages "sh" }}
+                                <div class="code-snippet js-code-block {{if eq $file_extension "sh"}} default {{end }} {{ if eq $count 0}}d-block{{else}}d-none{{end}} code-block-{{ $lang.pygments_name }}" >
+                                    <div class="code-button-wrapper position-absolute">
+                                        <button class="btn text-primary js-copy-button">Copy</button>
+                                    </div>
+                                    {{ $pathParams := $.Scratch.Get "pathParams" }}
+                                    <div class="highlight">
+                                        <pre class="chroma">
+                                            <code class="language-fallback" data-lang="fallback">
+{{- with $pathParams -}}
```

Comment:
Can you indent this block below to follow the rest of the code style?

Reply:
I am not sure how to make the inside code not indented.